Sources/PackageDescription/BuildSettings.swift

```diff
@@ -131,9 +131,9 @@ public struct CSetting: Encodable {
         return CSetting(name: "define", value: [settingValue], condition: condition)
     }
 
-    /// Set unsafe flags to pass arbitrary command-line flags to the corresponding build tool.
+    /// Sets unsafe flags to pass arbitrary command-line flags to the corresponding build tool.
     ///
-    /// As the usage of the word "unsafe" implies, the Swift Package Manager
```

Comment:
Is this right? Looking at swift.org, it seems like we are using the spelling "Swift Package Manager"

Reply:
Fixed. Now using title case.